### PR TITLE
Report image rendering fix

### PR DESCRIPTION
**Pull request by SchrodingersGat**
Adds ability to specify which image variation to use when rendering a "part_image" in a report.

If an uploaded image is very large this can cause the report size to bloat out.

- Can use "preview" image e.g. `{% part_image my_part preview=True %}`
- Can use "thumbnail" image  e.g. `{% part_image my_part thumbnail=True %}`
- Can also specify image width / height parameters

### CODE REVIEW

File: InvenTree/label/tests.py [hunk 1]
```diff
@@ -154,8 +154,6 @@ def test_print_part_label(self):
         self.assertIn(f"part: {part_pk} - {part_name}", content)
         self.assertIn(f'data: {{"part": {part_pk}}}', content)
         self.assertIn(f'http://testserver/part/{part_pk}/', content)
-        self.assertIn("img/blank_image.png", content)
-        self.assertIn("img/inventree.png", content)
```
Comment (matmair) [inline InvenTree/label/tests.py:158]: Question: Why are those tests removed?

File: InvenTree/report/templatetags/report.py [hunk 2]
```diff
@@ -184,13 +225,17 @@ def part_image(part: Part):
     Raises:
         TypeError if provided part is not a Part instance
     """
-    if type(part) is Part:
-        img = part.image.name
+    if type(part) is not Part:
+        raise TypeError("part_image tag requires a Part instance")
```
Comment (matmair) [inline InvenTree/report/templatetags/report.py:229]: Suggestion: This error is not translated
```suggestion
        raise TypeError(_("part_image tag requires a Part instance"))
```

**Review by matmair — APPROVED** [review]
Looks good in general;

File: InvenTree/label/tests.py (hunk 1 above)
Comment (SchrodingersGat) [reply InvenTree/label/tests.py:158]: Replacing these now, they are outdated with the new approach

**Pull request merged**